dcos_launch/config.py

```diff
@@ -293,7 +293,7 @@ def _validate_genconf_scripts(genconf_dir, dcos_config):
     'enable_selinux': {
         'type': 'boolean',
         'default_setter': lambda doc:
-            doc.get('dcos_version', '') >= '1.12' and 'dcos-enterprise' in doc['installer_url'] and
+            doc.get('dcos_version', 0) >= 1.12 and 'dcos-enterprise' in doc['installer_url'] and
```

Comment:
Is the purpose of this PR to be able to parse this version string correctly?

Reply:
Maybe we can use `parse_version` instead? https://setuptools.readthedocs.io/en/latest/pkg_resources.html#parsing-utilities

```
>>> parse_version('master') > parse_version('1.12')
False
>>> parse_version('1.12.1') > parse_version('1.12')
True
>>> parse_version('1.12') > parse_version('1.12')
False
>>> parse_version('1.12') >= parse_version('1.12')
True
>>>
```